libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -56,7 +57,8 @@ export class SuggestionsComponent implements OnInit, AfterViewInit {
     constructor(
         private suggestionsService: SuggestionsService,
         private dotLanguageService: DotLanguageService,
-        private cd: ChangeDetectorRef
+        private cd: ChangeDetectorRef,
+        public elementRef: ElementRef
```

Comment:
why?

Reply:
I needed a way to access the html of the component instance and add it to the tippy.